Sources/Core/API/TextureManager.swift

```diff
@@ -7,6 +7,12 @@
 import Foundation
 import CoreGraphics
 
+public enum ErrorMode {
```

Comment:
Would be great if you could provide a simple documentation comment for this type 👍